day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,18 @@
+number_of_frogs = 10
+
+
+number_of_frogs.times.each do | _ |
+  puts "#{number_of_frogs} speckled frogs sat on a log"
+  number_of_frogs = number_of_frogs - 1
+  puts "eating some most delicious bugs."
+  puts "One jumped in the pool where its nice and cool,"
+
+
+  if number_of_frogs == 0
+    puts "then there were no more speckled frogs!"
+  elsif number_of_frogs > 1
+    puts "then there were #{number_of_frogs} speckled frogs."
```

Comment:
Very nice conditional! You've done well to see that order matters here, and kept things pretty simple. I like just tracking the one variable, `number_of_frogs`, and you leverage WHEN you modify that variable to use it in different ways.